ament_yamllint/ament_yamllint/main.py

```diff
@@ -0,0 +1,275 @@
+#!/usr/bin/env python3
+
+# Copyright 2022 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+import errno
+import os
+import re
+import shutil
+import subprocess
+import sys
+import tempfile
+import time
+from xml.sax.saxutils import escape
+from xml.sax.saxutils import quoteattr
+
+import yaml
+
+
+def main(argv=sys.argv[1:]):
+    config_file = os.path.join(
+        os.path.dirname(__file__), 'configuration', 'yamllint.yaml')
+
+    extensions = ['yaml', 'yml']
+
+    parser = argparse.ArgumentParser(
+        description='Check YAML style using YAMLlint.',
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+    parser.add_argument(
+        '-c', '--config',
+        metavar='CFG',
+        default=config_file,
+        dest='config_file',
+        help='The config file')
+    parser.add_argument(
+        '--linelength', metavar='N', type=int,
+        help='The maximum line length (default: specified in the config file)')
+    parser.add_argument(
+        'paths',
+        nargs='*',
+        default=[os.curdir],
+        help='The files or directories to check. For directories files ending '
+             'in %s will be considered.' %
```

Comment:
```suggestion
        help='The files or directories to check. For each directory only files ending '
             'in %s will be considered.' %
```

(or something like that; having `directories files` looks weird)

Also, if you list a directory is it searched recursively?